pkgs/development/compilers/cudatoolkit/auto-add-opengl-runpath-hook.sh

```diff
@@ -1,14 +1,22 @@
-# Run autoOpenGLRunpath on all files
+# Run addOpenGLRunpath on all dynamically linked, ELF files
 echo "Sourcing auto-add-opengl-runpath-hook"
 
-autoAddOpenGLRunpathPhase  () {
-    # TODO: support multiple outputs
-    for file in $(find ${out,lib,bin} -type f); do
-        addOpenGLRunpath $file
-    done
-}
+autoAddOpenGLRunpathPhase() (
+  local outputPaths=($(for o in $(getAllOutputNames); do echo "${!o}"; done))
+  find "${outputPaths[@]}" -type f -executable -print0  | while IFS= read -rd "" f; do
+    if isELF "$f"; then
+      if patchelf --print-needed "$f" >/dev/null 2>&1; then
+        echo "autoAddOpenGLRunpathHook: patching $f"
+        addOpenGLRunpath "$f"
+      elif [ -n "${DEBUG-}" ]; then
+        echo "autoAddOpenGLRunpathHook: skipping ELF file $f"
+      fi
+    fi
+  done
+)
 
 if [ -z "${dontUseAutoAddOpenGLRunpath-}" ]; then
-    echo "Using autoAddOpenGLRunpathPhase"
-    postFixupHooks+=(autoAddOpenGLRunpathPhase)
+  echo "Using autoAddOpenGLRunpathPhase"
```

Comment:
Just a formatting change?

Reply:
My rewrite uses 2-space tabs so this is to make it consistent.